### PR TITLE
Fix assets without account's informations [Delivers #105026242]

### PR DESCRIPTION
**Pull request by ThadeuBrito**
@chiefGui @zegomesjf

### CODE REVIEW

File: src/components/live-displays/ThreeAssets/ThirdAsset.jsx [hunk 1]
```diff
@@ -30,18 +30,30 @@ export default class ThirdAsset extends Component {
   }
 
   renderAuthorAvatar() {
+    let avatar
+    if (this.props.asset.account) {
+      avatar = this.props.asset.account.avatar
+    } else {
+      avatar = 'http://beta.avatars.io/static/default_128.jpg'
```
Comment (chiefGui) [inline src/components/live-displays/ThreeAssets/ThirdAsset.jsx]: `https://`

**Comment by chiefGui** [comment]
:+1:

**Comment by zegomesjf** [comment]
:+1:

File: src/components/live-displays/ThreeAssets/ThirdAsset.jsx (hunk 1 above)
Comment (zegomesjf) [inline src/components/live-displays/ThreeAssets/ThirdAsset.jsx]: :+1:

**Pull request merged**